src/omt/omt_optimizer.cpp

```diff
@@ -67,6 +67,7 @@ Node OMTOptimizer::mkStrongIncrementalExpression(
     {
       if (targetType.isInteger() || targetType.isReal())
       {
+        Assert(!targetType.isReal());  // remove this if real support is added
```

Comment:
Thanks for this assertion, but i think that now it is very confusing because you have two assertions one after the other and the first subsumes the second. 

Since right now Reals are not supported, please just remove the Reals, like you do for FP.

It will be very easy to add these cases later, but for now including them only creates confusion.

Reply:
formatted